net/node/node.go

```diff
@@ -375,6 +377,22 @@ func (node *node) Xmit(message interface{}) error {
 	return nil
 }
 
+func (node *node) BroadcastTransaction(from Noder, txn *transaction.Transaction) error {
+	buffer, err := NewTxn(txn)
+	if err != nil {
+		log.Error("Error New Tx message: ", err)
+		return err
+	}
+	node.txnCnt++
+	for _, n := range node.GetSyncFinishedNeighbors() {
+		if n.GetRelay() && n.GetID() != from.GetID() {
+			n.Tx(buffer)
+		}
```

Comment:
What will happened if the node connection lost after get the syncfinishedneighbors list?